envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/OutgoingPermissionsTest.kt

```diff
@@ -57,8 +57,8 @@ abstract class OutgoingPermissionsTest : EnvoyControlTestConfiguration() {
             val unreachableDomainResponse = callDomain("www.another-example.com")
 
             // then
-            assertThat(unreachableResponse).isUnreachable().hasLocationHeaderFrom("not-accessible")
-            assertThat(unregisteredResponse).isUnreachable().hasLocationHeaderFrom("unregistered")
+            assertThat(unreachableResponse).isUnreachable()
+            assertThat(unregisteredResponse).isUnreachable()
```

Comment:
Removed because the location header is not sat any more: https://www.envoyproxy.io/docs/envoy/latest/intro/version_history#january-20-2020